mobile/src/main/java/rs/readahead/washington/mobile/mvp/presenter/TellaFileUploadSchedulerViewModel.kt

```diff
@@ -0,0 +1,76 @@
+package rs.readahead.washington.mobile.mvp.presenter
```

Comment:
Should we put this (and all other) models somewhere else?

Reply:
Yes, it should, I just fixed that.